src/Compilers/Core/SharedCollections/ArrayBuilder.cs

```diff
@@ -75,6 +75,10 @@ public int Count
             {
                 return _builder.Count;
             }
+            set
```

Comment:
The change uses it as Clip, for some reason Clip didn't catch my eye. Would you prefer me to undo this change?